### ISSUE TITLE
Text Font, Text Size, Text color, erase it are not changing according to the selection in post doubt forum.

### ISSUE THREAD

**Issue by Abhijay007** [codeforcauseorg/edu-client#399]
## Describe the bug

Features like Text font change, Text color, Text Size, Eraser are not changing accordingly to the selection in the post doubt forum. not changing in preview.

## To Reproduce

1. click on the forum
2. click on start discussion
3. enter text and select font, size, color in the given option
4. post discussion

## Expected behavior

changes according to the selection 

## Environment (please complete the following information):

- OS: Windows 10
- Browser: Chrome

## Additional context

![ezgif com-gif-maker](https://user-images.githubusercontent.com/64387054/132099354-e6cd1c07-c9bf-4329-b33f-361a72add78d.gif)

**Comment by Abhijay007** [comment]
can I work on it?

**Comment by adarsh-technocrat** [comment]
Sure @Abhijay007  go ahead :)

**Comment by adarsh-technocrat** [comment]
@Abhijay007  they do change the thing is you need to select the specific text that you want to change !

**Comment by adarsh-technocrat** [comment]
@Abhijay007  we have used [Draf.js](https://draftjs.org/) here you can quickly checkout  documentation

**Comment by Abhijay007** [comment]
> @Abhijay007 they do change the thing is you need to select the specific text that you want to change!

yes, it is working but I think that the "eraser feature" is not working even after selection. And thanks for pointing it out I will raise other issues soon.

**Comment by Abhijay007** [comment]
> @Abhijay007 we have used [Draf.js](https://draftjs.org/) here you can quickly checkout documentation

sure :)
